src/components/Header/style.css

```diff
@@ -120,7 +120,7 @@ header{
     background: #29ad5b;
     border: none;
 }
-.btn--add-product__text-reduction{
+.btn-type-secondary{
```

Comment:
According to the BEM official docs `block-name__element-name_modifier-name`.

Reply:
rename className